javascriptv3/example_code/transcribe/README.md

```diff
@@ -2,25 +2,42 @@
 Amazon Transcribe makes it easy for developers to add speech to text capabilities to their applications.
 
 ## Code examples
-This is a workspace where you can find the following AWS SDK for JavaScript v3 Transcribe examples:
-- [Create Transcribe job](src/transcribe_create_job.js)
-- [Create Transcribe medical job](src/transcribe_create_medical_job.js)
-- [Delete Transcribe job](src/transcribe_delete_job.js)
-- [Delete Transcribe medical job](src/transcribe_delete_medical_job.js)
-- [List Transcribe jobs](src/transcribe_list_jobs.js)
-- [List Transcribe medical jobs](src/transcribe_list_medical_jobs.js)
+This is a workspace where you can find the following AWS SDK for JavaScript v3 Amazon Transcribe examples:
+- [Create Transcribe job](src/transcribe_create_job.js) (StartTranscriptionJobCommand)
+- [Create Transcribe medical job](src/transcribe_create_medical_job.js) (StartMedicalTranscriptionJobCommand)
+- [Delete Transcribe job](src/transcribe_delete_job.js) (DeleteTranscriptionJobCommand)
+- [Delete Transcribe medical job](src/transcribe_delete_medical_job.js) (DeleteMedicalTranscriptionJobCommand)
+- [List Transcribe jobs](src/transcribe_list_jobs.js) (ListTranscriptionJobsCommand)
+- [List Transcribe medical jobs](src/transcribe_list_medical_jobs.js) (ListMedicalTranscriptionJobsCommand)
```

Comment:
All of these have to be 'Amazon Transcribe', per the service names guide ('Transcribe' alone is disallowed).

Reply:
fixed